src/styles/blocks/body.css

```diff
@@ -0,0 +1,4 @@
+.body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's recommended to style all elements using `.classes`. In this case, if `.body` is meant to style the `<body>` tag, you should use the tag name directly. If `.body` is a class for a different element, consider renaming it to better represent the content or purpose of that element.